screens/home/src/main/java/dev/sijanrijal/popularmovies/home/repository/HomeRepositoryImpl.kt

```diff
@@ -13,7 +13,7 @@ class HomeRepositoryImpl @Inject constructor(
         withContext(Dispatchers.IO) {
             val response = nowPlayingApi.getNowPlayingMovies()
             when {
-                response.isSuccessful -> Result.Success(response.body()!!)
+                response.isSuccessful -> Result.Success(response.body()!!.movies.filterNotNull())
```

Comment:
@sijanr   ⚠️  `!!` is risky 😉 

```suggestion
response.isSuccessful -> { 
response.body()?.let {
Result.Success(movies.filterNotNull())
} ?: run {
Result.Failure
}
}
```

Reply:
Thanks for reviewing. I'll make it null safe. 😄